### PR TITLE
Sms controls

### PR DESCRIPTION
**Pull request by msimon42**
### Sidebar Checklist

- [ ] Request reviewers
- [ ] Assign yourself and other contributors
- [ ] Link to project

### Issues Resolved
Resolves #

### Problem Addressed
Users had no way to get reminders
### Solution Implemented
Users can now get reminder texts and emails

### CODE REVIEW

File: app/jobs/notifications_worker.rb [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
-  def perform(contact_info, message)
-    phone_number = contact_info[:phone_number]
+  def perform(contact_info, message, type)
+    phone_number = contact_info[:phone_number].to_s
     address = contact_info[:email_address]
-    response = SmsService.new.send_sms(phone_number, message) if phone_number
-    if !phone_number || response['response'] != 'success'
-      AppointmentNotif.initial_message(address, message).deliver_now
-    end
+    sms = SmsService.new.send_sms(phone_number, message)
```
Comment (DanielEFrampton) [inline app/jobs/notifications_worker.rb]: Matt, does this still work without the check for the phone number to be present?

File: app/models/message_generator.rb [hunk 2]
```diff
@@ -5,6 +5,11 @@ def pairing_notification(name, date, time)
     "View it at #{paired_link}."
   end
 
+  def pairing_reminder(time)
+    "Friendly reminder that you have a pairing booked today at #{times[time]}. " +
```
Comment (DanielEFrampton) [inline app/models/message_generator.rb]: Can we change "today at" to "today from" for consistency with the other messages?

File: lib/tasks/scheduler.rake [hunk 3]
```diff
@@ -0,0 +1,11 @@
+desc 'Send reminders for all morning pairingd'
```
Comment (DanielEFrampton) [inline lib/tasks/scheduler.rake]: Small typo at end of string

File: app/models/date_formatter.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+class DateFormatter
+  def self.format(date)
+    dtime = date.to_date.asctime.gsub("00:00:00 ", "")
+    dtime[8] = '0' if dtime[8] == ' '
+    dtime
```
Comment (DanielEFrampton) [inline app/models/date_formatter.rb]: Nice. If you wanted to simplify this, the `strftime` method is really powerful and could more precisely spit out what you want without having to use gsub: https://ruby-doc.org/core-2.6.2/Time.html#method-i-strftime

File: app/jobs/notifications_worker.rb (hunk 1 above)
Comment (msimon42) [reply app/jobs/notifications_worker.rb]: Yes, it does work. I made some changes to the microservice.

File: app/models/message_generator.rb (hunk 2 above)
Comment (msimon42) [reply app/models/message_generator.rb]: Done.

File: lib/tasks/scheduler.rake (hunk 3 above)
Comment (msimon42) [reply lib/tasks/scheduler.rake]: Done.

**Pull request merged**